### PR TITLE
Adds Debian to nav (for uyuni only), edits.

### PR DESCRIPTION
**Pull request by Loquacity**
Following up on https://github.com/uyuni-project/uyuni-docs/pull/111

Note that I've added it to the nav, and put the Uyuni conditional at that level, rather than in individual sections in the chapter itself. This was for a few reasons:

* The SUMA content seemed incomplete
* It seems to me that to manage Debian clients on SUMA, you need to do at least a few things that are technically unsupported. We should probably avoid this kind of content where possible.
* It just makes everything simpler to have the conditional at a higher level (and we also intend to split the Ubuntu content in a similar way at a later date, too).

You will also note that I've commented out a couple of larger chunks of content late in the chapter, mostly because it's repeating stuff we have elsewhere in the docs suite  (CLM and bootstrapping). I'm happy to include it at a high level with an xref or something, instead.

### CODE REVIEW

File: modules/client-configuration/pages/clients-debian.adoc [hunk 1]
```diff
@@ -3,174 +3,119 @@
...
 This section contains information about registering Salt clients running {debian} operating systems.
 
-{productname} supports for {debian} 9 Stretch and 10 Buster clients is experimental.
+[IMPORTANT]
+====
+{suse} does not provide support for {debian} operating systems.
+{productname} allows you to manage {debian} clients, but support is not provided.
+Using {productname} to manage {debian} clients is experimental.
+These instructions have been tested on {debian}{nbsp}9 Stretch and {debian}{nbsp}10 Buster.
+Do not rely on them in a production environment.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-debian.adoc]: It's not that clear what "them" is here.

File: modules/client-configuration/pages/clients-debian.adoc [hunk 2]
```diff
@@ -3,174 +3,119 @@
...
 ----
 
-Ensure DNS is correctly configured and provides an entry for the client, otherwise configure the `/etc/hosts` files on both the uyuni server and the client with the appropriate entries.
-
-The client has to have NTP synchronised correctly with the Uyuni server before the registration
-====
+Ensure DNS is correctly configured and provides an entry for the client.
+Alternatively, you can configure the ``/etc/hosts`` files on both the {productname} Server and the client with the appropriate entries.
 
+The client must have NTP synchronized correctly with the {productname} server before registration.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-debian.adoc]: I'd just say: "... must have _time_ synchronized correctly."  We usually do not configure the server as an NTP time server.

File: modules/client-configuration/pages/clients-debian.adoc [hunk 3]
```diff
@@ -3,174 +3,119 @@
...
 
-ifeval::[{suma-content} == true]
-* In the {productname} {webui}, navigate to menu:Admin[Setup Wizard] and select the [guimenu]``SUSE Products`` tab.
-+
-This dialog displays a completion bar for each product when they are being synchronized.
-endif::[]
-ifeval::[{uyuni-content} == true]
-* In the {productname} {webui}, navigate to menu:Software[Manage > Channels], then click on the channel associated to the repository. Navigate to the [guimenu]``Repositories``, then [guimenu]``Sync`` and check [systemitem]``Sync Status``
-endif::[]
+* In the {productname} {webui}, navigate to menu:Software[Manage > Channels], and click on the channel associated to the repository.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-debian.adoc]:
```suggestion
* In the {productname} {webui}, navigate to menu:Software[Manage > Channels], and click the channel associated to the repository.
```

**Comment by gnuninu** [comment]
Hi Lana, thanks a lot great work!
FYI i have committed some changes to the page

Basically the suffix -uyuni is required since we have added the related entries in the channels.ini file specific for uyuni.
Suma should use the previous ones without the suffix. The code has been updated to support both suma and uyuni

```
For {debian} 9:

spacewalk-common-channels -u admin -a amd64-deb debian-9-pool-amd64-uyuni
spacewalk-common-channels -u admin -a amd64-deb debian-9-amd64-main-security-uyuni
spacewalk-common-channels -u admin -a amd64-deb debian-9-amd64-main-updates-uyuni
spacewalk-common-channels -u admin -a amd64-deb debian-9-amd64-uyuni-client-master

spacewalk-repo-sync --channel debian-9-pool-amd64-uyuni &
spacewalk-repo-sync --channel debian-9-amd64-main-security-uyuni &
spacewalk-repo-sync --channel debian-9-amd64-main-updates-uyuni &
spacewalk-repo-sync --channel debian-9-amd64-uyuni-client-master &

For {debian} 10:

spacewalk-common-channels -u admin -a amd64-deb debian-10-pool-amd64-uyuni
spacewalk-common-channels -u admin -a amd64-deb debian-10-amd64-main-security-uyuni
spacewalk-common-channels -u admin -a amd64-deb debian-10-amd64-main-updates-uyuni
spacewalk-common-channels -u admin -a amd64-deb debian-10-amd64-uyuni-client-master

spacewalk-repo-sync --channel debian-10-pool-amd64-uyuni &
spacewalk-repo-sync --channel debian-10-amd64-main-security-uyuni &
spacewalk-repo-sync --channel debian-10-amd64-main-updates-uyuni &
spacewalk-repo-sync --channel debian-10-amd64-uyuni-client-master-uyuni &

**Comment by Loquacity** [comment]
> Hi Lana, thanks a lot great work!
> FYI i have committed some changes to the page
> 
Is there a PR for me to review?

**Comment by gnuninu** [comment]
> Is there a PR for me to review?

sure
https://github.com/uyuni-project/uyuni/pull/1917

**Comment by Loquacity** [comment]
> > Is there a PR for me to review?
> 
> sure
> [uyuni-project/uyuni#1917](https://github.com/uyuni-project/uyuni/pull/1917)

Heh, I meant a docs PR :wink: All good, I did it here: https://github.com/uyuni-project/uyuni-docs/pull/125

**Comment by gnuninu** [comment]
ops sorry :) i only changed the commands above nothing else. 

>  I did it here: #125 
Thanks!

**Pull request merged**